### PR TITLE
Deps/update ott

### PR DESCRIPTION
**Pull request by MUCDK**

### CODE REVIEW

**Comment by giovp** [comment]
I think this can be closed for now?

File: src/moscot/backends/ott/solver.py [hunk 1]
```diff
@@ -141,7 +141,7 @@ def __init__(
         self,
         jit: bool = True,
         rank: int = -1,
-        epsilon: float = 0.0,
+        epsilon: Optional[float] = 0.0,
```
Comment (michalk8) [inline src/moscot/backends/ott/solver.py:144]: This cannot be optional for LR solvers.

File: tests/problems/space/test_mixins.py [hunk 2]
```diff
@@ -51,6 +51,7 @@ def test_analysis(self, adata_space_rotate: AnnData):
                     == adata_ref[adata_ref.obs.batch == c].obsm["spatial"].shape
                 )
 
+    @pytest.mark.skip("TODO(giovp): regenerate solutions")
```
Comment (michalk8) [inline tests/problems/space/test_mixins.py:54]: These should be regenerated now.

**Review by michalk8 — CHANGES_REQUESTED** [review]
Looks good, just minor comments.

**Pull request merged**